ParselyTracker/EventQueue.swift

```diff
@@ -20,6 +20,11 @@ struct EventQueue<T> {
         os_log("Event pushed into queue", log: OSLog.tracker, type: .debug)
         list.append(element)
     }
+
+    mutating func push<Collection>(contentsOf elements:Collection) where T == Collection.Element, Collection: Sequence {
```

Comment:
A new test for this function would help us catch possible future regressions.

Reply:
Thank you kindly for the review! The new test has been added.